Zend/tests/024.phpt

```diff
@@ -41,17 +41,17 @@ NULL
 Warning: Undefined variable $b in %s on line %d
 int(1)
 
-Warning: Trying to get property '1' of non-object in %s on line %d
+Warning: Attempt to read property '1' on int in %s on line %d
```

Comment:
I still believe that the get/set wording should be kept unchanged at least in order to correspond to the names of the magic `__get()` and `__set()` methods.

Reply:
For me, the `read` terminology still seems fine since the description of `__get()` literally says that it's used for reading data. Besides, using `isset()` on a non-object doesn't provoke this error, so it also won't cause any issues. I can change the wording to `get` though if Nikita agrees, but I'd support `read` at this point.